test/unit/transports/local_test.rb

```diff
@@ -6,11 +6,12 @@
 class TransportHelper
   attr_accessor :transport
 
-  def initialize(type = :mock)
+  def initialize(user_opts = {})
+    opts = {platform_name: 'mock', family_hierarchy: ['mock']}.merge(user_opts)
     Train::Platforms::Detect::Specifications::OS.load
-    plat = Train::Platforms.name('mock')
+    plat = Train::Platforms.name(opts[:platform_name])
+    plat.family_hierarchy = opts[:family_hierarchy]
```

Comment:
Does `plat.family_hierarchy` need to be an array? The default you have set makes me believe that maybe that's true...

If so, can I suggest changing this to:

```ruby
plat.family_hierarchy = Array(opts[:family_hierarchy])
```

... so that it's always an array, even if a future test author supplies a single family string?

Reply:
Based on a `grep` I did I see it defined as an array, but on the `mock` transport I see we do a flatten.

https://github.com/chef/train/blob/master/lib/train/transports/mock.rb#L80

I think I'll do what you said, but leave my examples as single item arrays.

Thoughts?